components/ui/main.tsx

```diff
@@ -0,0 +1,177 @@
+import Image from 'next/image'
+import Dashboard from '../../public/images/dashboard.png'
+import custom from '../../public/images/custom.png'
+import { CheckIcon, DoubleArrowRightIcon } from '@radix-ui/react-icons'
+
+export default function Main() {
+  return (
+    <section className="body-font text-gray-600">
+      <div className=" animate-trans-up mx-auto max-w-5xl pb-24 pt-52">
+        <h1 className="font-4 lh-6 ld-04 mb-6 text-center font-bold text-white lg:text-4xl">
+          Getting your customer reviews by only 1 Link
+        </h1>
+        <h2 className="font-4 lh-6 ld-04 pb-11 text-center font-semibold text-gray-700 lg:text-2xl">
+          Setting up a Review system can be quite tedious. Review App provides a
+          <br />
+          seamless and hassle free way to acquire feedback on your products.
+        </h2>
+        <div className="ml-6 flex flex-row items-center justify-center gap-10 text-center">
+          <div className="text-md focus:shadow-outline inline-flex transform animate-bounce items-center rounded-xl bg-transparent bg-white px-7 py-3 font-semibold text-black transition  md:mt-0">
+            <div className="flex  text-lg">
+              <span className=" justify-center text-purple-500">
+                Get Started
+              </span>
+            </div>
+          </div>
+          <div className="text-md  flex cursor-pointer items-center justify-center rounded-xl bg-transparent bg-gradient-to-r from-purple-500 to-purple-800 px-14 py-3 font-semibold tracking-tighter text-white  transition-all duration-200 ease-in-out hover:scale-125 md:mt-0">
+            <div className="flex text-center text-lg ">
```

Comment:
Make a signup to navigate to `app.localhost:3000/signup` and take these values from process.env